test/sol5/kyberNetwork.js

```diff
@@ -26,6 +26,7 @@ const maxDestAmt = new BN(2).pow(new BN(255));
 const minConversionRate = new BN(0);
 const oneEth = new BN(10).pow(ethDecimals);
 const defaultNetworkFeeBps = new BN(25);
+const KYBER_MAX_QTY = new BN(10).pow(new BN(28)); // 10B tokens
```

Comment:
please take this value from helper.js.
add it to line 17.

Reply:
resolved in 8a85cb2840731bfd6de26d589980f04b587331c4